packages/concerto-vocabulary/lib/vocabulary.js

```diff
@@ -72,6 +65,37 @@ class Vocabulary {
         return this.content.namespace;
     }
 
+    /**
+     * Validates a locale
+     * @param {string} locale the locale to validate
+     * @throws {Error} if the locale is invalid
+     * @private
```

Comment:
Why private? I thought we wanted to expose this.

Reply:
updated